src/qml/pages/node/NodeRunner.qml

```diff
@@ -18,26 +18,9 @@ Page {
     ColumnLayout {
         spacing: 0
         anchors.fill: parent
-        ColumnLayout {
-            width: 600
-            spacing: 0
-            anchors.centerIn: parent
-            Component.onCompleted: nodeModel.startNodeInitializionThread();
-            Image {
-                Layout.alignment: Qt.AlignCenter
-                source: "image://images/app"
-                sourceSize.width: 64
-                sourceSize.height: 64
-            }
-            BlockCounter {
-                Layout.alignment: Qt.AlignCenter
-                blockHeight: nodeModel.blockTipHeight
-            }
-            ProgressIndicator {
-                width: 200
-                Layout.alignment: Qt.AlignCenter
-                progress: nodeModel.verificationProgress
-            }
-        }
+
+        Component.onCompleted: nodeModel.startNodeInitializionThread();
+
+        BlockClockComponent{}
```

Comment:
```
GUI: qrc:/qml/pages/node/NodeRunner.qml:24:9: QML BlockClockComponent: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
```
Should be able to remove the ColumnLayout containing the BlockClockComponent and just define the BlockClockComponent as the content to this Page.